app/assets/stylesheets/global.scss

```diff
@@ -15,6 +15,6 @@
   font-weight: bold;
 }
 
-.memory{
+#your_memories .memory {
```

Comment:
Avoid using id selectors<br>Selector `your_memories` should be written in lowercase with hyphens